src/index.html

```diff
@@ -3,12 +3,34 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="logo.png"
+        alt="logo"
+        class="logo__img"
+      >
+      <nav class="nav">
+        <ul class="nav__links">
+          <li><a class="nav__link is-active" href="#apple">Apple</a></li>
```

Comment:
You need to fix with code style
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">Apple</a>
          </li>
```